code/modules/projectiles/guns/energy/energy.dm

```diff
@@ -77,16 +102,24 @@
 	shaded_charge = TRUE
 	can_holster = TRUE
 	execution_speed = 5 SECONDS
+	extended_description = "The EG-14 is Shellguard Munitions' premium offering in the energy weapons market. It is equal to or superior to the EG-7 in every way, boasting the same fire rate, \
+	stopping power, and cell capacity, whilst also being better suited as an improvised melee weapon thanks to superior ergonomics and weight distribution - all offered in a pistol-sized form factor.<br><br>\
+	The revolver's cylinder does indeed rotate with every shot, and it can be spun just like the cylinder of a ballistic revolver - although this has no bearing on the actual functionality of the weapon, \
+	and is included purely as a stylistic embellishment.<br><br>\
+	The price of the EG-14 puts it out of reach of a large portion of the market, but it sells very well among bodyguards and bountyhunters."
 
 /obj/item/gun/energy/gun/blueshield/pdw9
 	name = "\improper PDW-9 energy pistol"
-	desc = "A military grade sidearm, used by many militia forces throughout the local sector."
+	desc = "A military-grade energy pistol favoured by mercenaries and militia forces. The fire selector has two settings: 'disable', and 'kill'."
```

Comment:
```suggestion
	desc = "A military-grade energy pistol favored by mercenaries and militia forces. The fire selector has two settings: 'disable', and 'kill'."
```
Another description that has been touched used the american version and it is the most used for this word on the codebase.